src/Platform.Handlers/samples/Sample.iOS/AppDelegate.cs

```diff
@@ -11,7 +15,11 @@ namespace Sample.iOS
 	// The UIApplicationDelegate for the application. This class is responsible for launching the
 	// User Interface of the application, as well as listening (and optionally responding) to application events from iOS.
 	[Register("AppDelegate")]
+#if !NET6_0
 	public class AppDelegate : FormsApplicationDelegate, IUIApplicationDelegate
+#else
```

Comment:
Commenting this out for now with NET6 targets because `Platform.iOS` is built against XA and not net6-ios

Once `Platform.iOS` is built against net6-ios then we can go back to FormsApplicationDelegate